src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -37,11 +38,13 @@ internal class HttpTriggerAttributeBindingProvider : ITriggerBindingProvider
     {
         internal const string HttpQueryKey = "Query";
         internal const string HttpHeadersKey = "Headers";
+        internal const string IdentitiesKey = "IdentitiesArray";
```

Comment:
Just call this this "Identities" to match other keys. E.g. for headers, we don't indicate collection type.